ssl/statem/statem_lib.c

```diff
@@ -278,10 +278,10 @@ MSG_PROCESS_RETURN tls_process_cert_verify(SSL *s, PACKET *pkt)
     EVP_PKEY *pkey = NULL;
     const unsigned char *data;
 #ifndef OPENSSL_NO_GOST
-    unsigned char *gost_data = NULL;
+    unsigned char *gost_data = NULL;
```

Comment:
Hmm, just noticed Travis complains here. Seems like it's a hard space, would you mind just rewriting those spaces?